release-notes/opensearch-dashboards.release-notes-2.2.0.md

```diff
@@ -0,0 +1,40 @@
+## Version 2.2.0 Release Notes
+
+#### Deprecations
+* Deprecate the Blacklist / Whitelist nomenclature ([#1808](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1808))
+* Alias OUI onto EUI ([#2080](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2080))
```

Comment:
Does this factor more into feature as we are consuming OUI and then it's Aliasing to EUI.

Reply:
will move to feature